src/main/resources/org.graylog.plugins.graylog-plugin-slack/graylog-plugin.properties

```diff
@@ -0,0 +1,12 @@
+# The plugin version
+version=${project.version}
+
+# The required Graylog server version
+graylog.version=${graylog.version}
+
+# When set to true (the default) the plugin gets a separate class loader
+# when loading the plugin. When set to false, the plugin shares a class loader
+# with other plugins that have isolated=false.
+#
+# Do not disable this unless this plugin depends on another plugin!
+isolated=false
```

Comment:
You are correct, it should be set to true as it does not depend on any other plugins